OrbitLinuxTracing/PerfEventOpen.h

```diff
@@ -100,6 +105,8 @@ static constexpr uint64_t SAMPLE_REGS_USER_ALL =
 //  some setting.
 static constexpr uint16_t SAMPLE_STACK_USER_SIZE = 65000;
 
+static constexpr uint16_t SAMPLE_STACK_USER_SIZE_8BYTES = 8;
```

Comment:
Should this be the size of a pointer and not an explicit 8?

Reply:
I'm undecided, as the other code that is related to this all assumes 8-byte registers (even the name of the constant). So what I did instead is adding `static_assert(sizeof(void*) == 8)`, at least for clarity.